pkg/plugin/framework/backup_item_action_server.go

```diff
@@ -14,126 +14,21 @@ See the License for the specific language governing permissions and
-	return &BackupItemActionPlugin{
-		pluginBase: newPluginBase(options...),
-	}
-}
-
-//////////////////////////////////////////////////////////////////////////////
-// client code
-//////////////////////////////////////////////////////////////////////////////
-
-// GRPCClient returns a clientDispenser for BackupItemAction gRPC clients.
-func (p *BackupItemActionPlugin) GRPCClient(c *grpc.ClientConn) (interface{}, error) {
-	return newClientDispenser(p.clientLogger, c, newBackupItemActionGRPCClient), nil
-}
-
-// BackupItemActionGRPCClient implements the backup/ItemAction interface and uses a
-// gRPC client to make calls to the plugin server.
-type BackupItemActionGRPCClient struct {
-	*clientBase
-	grpcClient proto.BackupItemActionClient
-}
-
-func newBackupItemActionGRPCClient(base *clientBase, clientConn *grpc.ClientConn) interface{} {
-	return &BackupItemActionGRPCClient{
-		clientBase: base,
-		grpcClient: proto.NewBackupItemActionClient(clientConn),
-	}
-}
-
-func (c *BackupItemActionGRPCClient) AppliesTo() (velero.ResourceSelector, error) {
-	res, err := c.grpcClient.AppliesTo(context.Background(), &proto.AppliesToRequest{Plugin: c.plugin})
-	if err != nil {
-		return velero.ResourceSelector{}, err
-	}
-
-	return velero.ResourceSelector{
-		IncludedNamespaces: res.IncludedNamespaces,
-		ExcludedNamespaces: res.ExcludedNamespaces,
-		IncludedResources:  res.IncludedResources,
-		ExcludedResources:  res.ExcludedResources,
-		LabelSelector:      res.Selector,
-	}, nil
-}
-
-func (c *BackupItemActionGRPCClient) Execute(item runtime.Unstructured, backup *api.Backup) (runtime.Unstructured, []velero.ResourceIdentifier, error) {
-	itemJSON, err := json.Marshal(item.UnstructuredContent())
-	if err != nil {
-		return nil, nil, err
-	}
-
-	backupJSON, err := json.Marshal(backup)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	req := &proto.ExecuteRequest{
-		Plugin: c.plugin,
-		Item:   itemJSON,
-		Backup: backupJSON,
-	}
-
-	res, err := c.grpcClient.Execute(context.Background(), req)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	var updatedItem unstructured.Unstructured
-	if err := json.Unmarshal(res.Item, &updatedItem); err != nil {
-		return nil, nil, err
-	}
-
-	var additionalItems []velero.ResourceIdentifier
-
-	for _, itm := range res.AdditionalItems {
-		newItem := velero.ResourceIdentifier{
-			GroupResource: schema.GroupResource{
-				Group:    itm.Group,
-				Resource: itm.Resource,
-			},
-			Namespace: itm.Namespace,
-			Name:      itm.Name,
-		}
-
-		additionalItems = append(additionalItems, newItem)
-	}
-
-	return &updatedItem, additionalItems, nil
-}
-
-//////////////////////////////////////////////////////////////////////////////
-// server code
-//////////////////////////////////////////////////////////////////////////////
-
 // GRPCServer registers a BackupItemAction gRPC server.
 func (p *BackupItemActionPlugin) GRPCServer(s *grpc.Server) error {
```

Comment:
per previous comment, this method is part of the `BackupItemActionPlugin` type which I'd probably opt to move into its own file rather than having it exist across the client and server files.